pkgs/os-specific/linux/kernel/linux-testing-bcachefs.nix

```diff
@@ -32,6 +32,13 @@
         sha256 = diffHash;
       };
 
-      extraConfig = "BCACHEFS_FS m";
+      extraConfig = ''
+        BCACHEFS_FS m
+        BCACHEFS_QUOTA y
+        BCACHEFS_POSIX_ACL y
+        # Small overhead in exchange of making it easier to debug random deadlocks and hangs
+        # For now, it seems a good investment.
+        BCACHEFS_DEBUG_TRANSACTIONS y
```

Comment:
Isn't it [enabled](https://evilpiepirate.org/git/bcachefs.git/tree/fs/bcachefs/Kconfig) by default?

```
config BCACHEFS_DEBUG_TRANSACTIONS
	bool "bcachefs runtime info"
	depends on BCACHEFS_FS
	default y
```

Reply:
You're definitely right… !